developer-docs-site/docs/tutorials/first-transaction-sdk.md

```diff
@@ -14,6 +14,8 @@ This tutorial introduces the Aptos SDKs and how to generate, submit, and verify
 ## Step 1: Pick an SDK
 
 * [Official Aptos Python SDK][python-sdk]
+* Official Aptos Typescript SDK -- TBA
```

Comment:
We don't have one?

Reply:
there's no proper site yet. so leaving for @banool